tests-integration/src/tests/factory.rs

```diff
@@ -2449,6 +2452,7 @@ fn run_test_stable_pool_swap_request(factory_chain_id: &str, router_chain_id: &s
                     refund_address: None,
                     forwarding_message: None,
                     vcoin_msg: None,
+                    unsafe_refund_voucher_to_recipient: None,
```

Comment:
Can we add a test case where this variable is set to `true`? Since `None` always defaults to `false`